robolectric/src/test/java/org/robolectric/shadows/ShadowHandlerTest.java

```diff
@@ -74,7 +73,8 @@ public void testDefaultConstructorUsesDefaultLooper() throws Exception {
   }
 
   private static Looper newLooper(boolean canQuit) {
-    return ReflectionHelpers.callConstructor(Looper.class, from(boolean.class, canQuit));
+    return ReflectionHelpers.callConstructor(
+        Looper.class, ReflectionHelpers.ClassParameter.from(boolean.class, canQuit));
```

Comment:
nit: can you change all of these to just `ClassParameter.from`? So that would require importing `import org.robolectric.util.ReflectionHelpers.ClassParameter`. That's the most common pattern in Robolectric.

Reply:
Done.